ci/deploy/remote/branchctl.sh

```diff
@@ -14,7 +14,7 @@ if [[ "$action" == "stop" ]]; then
 fi
 
 cp ./server-settings.json server/
-docker run -d --rm --name="skymp-server-$branch" --network=host \
+docker run -d --restart=always --rm --name="skymp-server-$branch" --network=host \
```

Comment:
I think it's better to use `unless-stopped`

Reply:
I want restart on error